src/clone.rs

```diff
@@ -163,6 +179,21 @@ macro_rules! to_return_value {
 /// closure(3);
 /// ```
 ///
+/// ### Renaming variables
+///
+/// ```
+/// use glib::clone;
+/// use std::rc::Rc;
+///
+/// let v = Rc::new(1);
+/// let u = Rc::new(2);
+/// let closure = clone!(@strong v as y, @weak u => move |x| {
```

Comment:
@GuillaumeGomez  can you add test with two @strong renamed variable too?

Reply:
What would it change?

Reply:
Just to show what syntax used in this case

Reply:
I'm not sure to follow...

Reply:
I was just not sure if this case work or need two `@strong`, now you can ignore this comment
```
clone!(@strong a as b, c as d => ...)
```

Reply:
The example you wrote doesn't work either: you have to add `@strong` or `@weak` before each variable.

Reply:
@EPashkin anything missing here?

Reply:
IMHO all good

Reply:
except for new merge conflict 😢